draft-ietf-quic-multipath.md

```diff
@@ -1312,10 +1312,8 @@ paths allows.
 An implementation should take care to handle different PMTU sizes across
 multiple paths. As specified in {{Section 14.3 of QUIC-TRANSPORT}} the
 DPLPMTUD Maximum Packet Size (MPS) is maintained for each combination of local and remote IP addresses.
-One simple option, if the PMTUs are relatively similar, is to apply the minimum PMTU of all paths to
-each path. The benefit of such an approach is to simplify retransmission
-processing as the content of lost packets initially sent on one path can be sent
-on another path without further frame scheduling adaptations.
+One simple option, if the PMTUs are similar, is to apply the minimum PMTU of all paths to
+each path.
```

Comment:
We have a separate issue for this #562 and I guess are still discussion it. Either we keep this PR open until the issues is resolved or we move this chance to a separate PR.

Reply:
Okay, reverting removing this sentence as this is addressed by PR #578 now to be able to merge this editorial PR now.
```suggestion
One simple option, if the PMTUs are similar, is to apply the minimum PMTU of all paths to
each path. The benefit of such an approach is to simplify retransmission
processing as the content of lost packets initially sent on one path can be sent
on another path without further frame scheduling adaptations.
```